pkg/services/librarypanels/database.go

```diff
@@ -105,6 +154,33 @@ func (lps *LibraryPanelService) getAllLibraryPanels(c *models.ReqContext) ([]Lib
 	return libraryPanels, err
 }
 
+// getConnectedDashboards gets all dashboards connected to a Library Panel.
+func (lps *LibraryPanelService) getConnectedDashboards(c *models.ReqContext, uid string) ([]int64, error) {
+	connectedDashboardIds := make([]int64, 0)
+	err := lps.SQLStore.WithDbSession(context.Background(), func(session *sqlstore.DBSession) error {
+		panel, err := getLibraryPanel(session, uid, c.SignedInUser.OrgId)
+		if err != nil {
+			return err
+		}
+
+		libraryPanelDashboards := make([]LibraryPanelDashboard, 0)
```

Comment:
Nit, but I think there's no need to create the array, only to declare it.
```suggestion
		var libraryPanelDashboards []LibraryPanelDashboard
```

Reply:
Great catch, thank you, and I've fixed it now!